core/src/crds.rs

```diff
@@ -160,6 +161,11 @@ impl Crds {
         self.table.get(label)
     }
 
+    pub fn get_contact_info(&self, pubkey: &Pubkey) -> Option<&ContactInfo> {
+        let label = CrdsValueLabel::ContactInfo(*pubkey);
+        self.table.get(&label)?.value.contact_info()
```

Comment:
Can be simplified to `self.lookup(&label)?.contact_info()`

Reply:
I feel like the extra level of indirection is unnecessary here, and this is already a simple one-liner and maybe even more explicit.